packages/integrations/vercel/README.md

```diff
@@ -129,6 +129,30 @@ export default defineConfig({
 });
 ```
 
+### Vercel Middleware
+
+You can use Vercel middleware to intercept a request and redirect before sending a response. Vercel middleware can run for Edge, SSR, and Static deployments. You don't need to install `@vercel/edge` to write middleware, but you do need to install it to use features such as geolocation. For more information see [Vercel’s middleware documentation](https://vercel.com/docs/concepts/functions/edge-middleware).
+
+1. Add a `middleware.js` file to the root of your project:
+
+    ```js
+    // middleware.js
+    export const config = {
+      // Only run the middleware on the admin route
+      matcher: '/admin',
```

Comment:
is there a better path we could use here? `/admin` is a peculiar route to be using